service/docker.go

```diff
@@ -583,17 +583,26 @@ func stopVideoContainer(ctx context.Context, cli *client.Client, requestId uint6
 	select {
 	case <-doesNotExist:
 	case <-notRunning:
-		removeContainer(ctx, cli, requestId, containerId)
+		removeContainer(ctx, cli, requestId, containerId, env)
 		return
 	case <-time.After(env.SessionDeleteTimeout):
-		removeContainer(ctx, cli, requestId, containerId)
+		removeContainer(ctx, cli, requestId, containerId, env)
 		return
 	}
 	log.Printf("[%d] [STOPPED_VIDEO_CONTAINER] [%s]", requestId, containerId)
 }
 
-func removeContainer(ctx context.Context, cli *client.Client, requestId uint64, id string) {
+func removeContainer(ctx context.Context, cli *client.Client, requestId uint64, id string, env Environment) {
 	log.Printf("[%d] [REMOVING_CONTAINER] [%s]", requestId, id)
+
+	if env.GracefulShutdown {
+		timeout := 10
+		err := cli.ContainerStop(ctx, id, ctr.StopOptions{Timeout: &timeout})
```

Comment:
stop container with specified timeout - 10 (default value, but I should specify it, otherwise container not stop correctly). Maybe I should reduce timeout to 5. In my test runs even 1 second is enough to stop my browser container correctly.

Reply:
I decided to reduce it to 5. it seems that this value is quite enough to cleanup